extensions/UserProfile/template/en/default/pages/user_profile.html.tmpl

```diff
@@ -108,6 +108,13 @@
       [% END %]
     </td>
   </tr>
+[% ELSE %]
+  <tr>
+    <th>Get Permissions</th>
+    <td>
```

Comment:
I think I tested it before creating the pr. But I will test it once more and let you know in irc